source/guide_minio.rst

```diff
@@ -0,0 +1,152 @@
+.. author:: Marius Bertram <marius@brtrm.de>
+
+.. tag:: file-storage
+.. tag:: lang-go
+
+.. highlight:: console
+
+.. sidebar:: MinIO
+
+  .. image:: _static/images/minio_logo.png
+      :align: center
+
+#####
+MinIO
+#####
+
+.. tag_list::
+
+MinIO_ is an open source object storage server,
+compatible with Amazon S3 writen in go.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Web Backends <web-backends>`
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`domains <web-domains>`
+
+License
+=======
+
+MinIO is licensed under Apache-2.0
+
+     https://github.com/minio/minio#license
+
+
+Prerequisites
+=============
+Download the latest MinIO binary into your bin folder and make it executable:
+
+.. code-block:: console
+
+    [isabell@stardust ~]$ wget https://dl.min.io/server/minio/release/linux-amd64/minio -O /home/isabell/bin/minio
+    [isabell@stardust ~]$ chmod +x /home/isabell/bin/minio
+    [isabell@stardust ~]$
+
+Create a folder for your data:
+
+.. code-block:: console
+
+    [isabell@stardust ~]$ mkdir /home/isabell/minio
+    [isabell@stardust ~]$
+
+.. include:: includes/web-backend.rst
+
+.. note:: MinIO default port is 9000
+
+Setup config
+============
+
+Run Minio once to get the master AccessKey and SecretKey:
+
+.. code-block:: console
+
+    [isabell@stardust ~]$ /home/isabell/bin/minio server /home/isabell/minio
+    [...]
+    AccessKey: 2MN0QSZJS104ROKHYKNC
+    SecretKey: rozZqplprT+PZrYb4Mr5GNxeeh+CegYiMBvrvwfD
+    [...]
+    [isabell@stardust ~]$
+
+.. note:: The AccessKey and the SecretKey are generated randomly. You can also find the keys in the MinIO config. The config file is located in the folder ``~/minio/.minio.sys/config/config.json``
+
+You can set your own AccessKey and SecretKey via ENVIROMENT VARIABLES:
+
+.. code-block:: console
+
+    [isabell@stardust ~]$ export MINIO_ACCESS_KEY=minio
+    [isabell@stardust ~]$ export MINIO_SECRET_KEY=miniostorage
+    [isabell@stardust ~]$
+
+Setup Supervisord
+=================
+Create the configuration ``~/etc/services.d/minio.ini``:
+
+.. code-block:: ini
+
+    [program:minio]
+    command=~/bin/minio server ~/minio
```

Comment:
This should work with `command=minio (...)`, because it's in the `$PATH`.

Reply:
`~` doesn't work here, please use `%(ENV_HOME)s/minio` as the parameter.

Reply:
> `~` doesn't work here, please use `%(ENV_HOME)s/minio` as the parameter.

`~` does work just checked it on biela

Reply:
I just checked it on machholz :sweat_smile: all other guides use `$(ENV_HOME)s`, so it seems nicer for consistency's sake in any case.